### PR TITLE
[stable/kube-state-metrics] Add imagePullSecrets in serviceAccount

### PR DESCRIPTION
**Pull request by tbuchier**
Signed-off-by: Tanguy Buchier <tanguy.buchier@gmail.com>

#### What this PR does / why we need it:

Add the ability to specify the imagePullSecrets field in the serviceAccount
Allow you to specify the serviceAccount name even if rbac is enabled

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by tbuchier** [comment]
need to bump version, because another change was added.

File: stable/kube-state-metrics/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -8,4 +9,7 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
   name: {{ template "kube-state-metrics.fullname" . }}
+imagePullSecrets:
+{{ toYaml .Values.serviceAccount.imagePullSecrets | indent 2 }}
 {{- end -}}
+{{- end -}}
```
Comment (cpanato) [inline stable/kube-state-metrics/templates/serviceaccount.yaml]: new line 😄

File: stable/kube-state-metrics/templates/_helpers.tpl [hunk 2]
```diff
@@ -23,3 +23,14 @@ If release name contains chart name it will be used as a full name.
...
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "kube-state-metrics.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "kube-state-metrics.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
```
Comment (cpanato) [inline stable/kube-state-metrics/templates/_helpers.tpl]: new line

File: stable/kube-state-metrics/Chart.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 0.14.1
+version: 0.14.2
```
Comment (cpanato) [inline stable/kube-state-metrics/Chart.yaml]:
```suggestion
version: 0.15.0
```

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/kube-state-metrics/templates/serviceaccount.yaml (hunk 1 above)
Comment (tbuchier) [reply stable/kube-state-metrics/templates/serviceaccount.yaml]: Done :slightly_smiling_face:

File: stable/kube-state-metrics/Chart.yaml (hunk 3 above)
Comment (tbuchier) [reply stable/kube-state-metrics/Chart.yaml]: Done

**Review by cpanato — APPROVED** [review]

**Pull request merged**